Description: Look for audacious-panel before audacious when looking up icons in
 the icon theme.
Author: William Pitcock <nenolod@atheme.org>
Origin: upstream, http://hg.atheme.org/audacious-plugins/audacious-plugins/rev/822c842d471e

diff -r 434206be39cf -r 822c842d471e src/statusicon/si_ui.c
--- a/src/statusicon/si_ui.c	Sat Mar 20 20:48:30 2010 -0500
+++ b/src/statusicon/si_ui.c	Wed Apr 14 08:23:37 2010 -0500
@@ -60,8 +60,9 @@
     GtkIconTheme *theme;
 
     theme = gtk_icon_theme_get_default();
-    if (gtk_icon_theme_has_icon(theme, "audacious"))
-    {
+    if (gtk_icon_theme_has_icon(theme, "audacious-panel")) {
+        icon = gtk_status_icon_new_from_icon_name("audacious-panel");
+    } else if (gtk_icon_theme_has_icon(theme, "audacious")) {
         icon = gtk_status_icon_new_from_icon_name("audacious");
     } else {
         icon = gtk_status_icon_new_from_file(DATA_DIR "/images/audacious_player.xpm");
